ncclient/transport/ssh.py

```diff
@@ -545,60 +548,43 @@ def run(self):
         def start_delim(data_len): return '\n#%s\n'%(data_len)
 
         try:
+            s = selectors.DefaultSelector()
+            s.register(chan, selectors.EVENT_READ)
+            logger.debug('selector type = %s', s.__class__.__name__)
             while True:
-                # select on a paramiko ssh channel object does not ever return it in the writable list, so channels don't exactly emulate the socket api
-                r, w, e = select([chan], [], [], TICK)
-                # will wakeup evey TICK seconds to check if something to send, more if something to read (due to select returning chan in readable list)
-                if r:
+
+                # Log what netconf:base version we are using this time
+                # round the loop; _base is updated when we receive the
+                # server's capabilities.
+                logger.debug('Currently selected netconf:base:%s', self._base)
```

Comment:
This message gets spammed to the logs once per TICK. Oops. I'll fix this in a PR coming shortly.